dashboard/src/pages/EventVolunteers.vue

```
@@ -103,10 +103,10 @@ const session = inject('$session')
 
 const route = useRoute()
 
-const isLead = () => {
+const isCoreTeam = () => {
     let currentUser = event.doc.event_members.filter((m) => m.email === session.user)
```

ESLint

[error] Delete `··`

(prettier/prettier)